packages/adapter-knex/lib/adapter-knex.js

```diff
@@ -143,7 +142,6 @@ class KnexAdapter extends BaseKeystoneAdapter {
     // Create an adjacency table for the (many to many) relationship field adapter provided
     const dbAdapter = this;
     try {
-      console.log(`Dropping table ${tableName}`);
```

Comment:
I know these logs are a bit annoying when running tests, but they're essential during a deployment, so let's leave them in.

Reply:
Can we then remove them from tests only?
With something like this
```
if (process.env.NODE_ENV !== 'test') {
  console.log(`Dropping table ${tableName}`);
}
```

Reply:
Yep, I think finding a way to silence them in tests would be a worthwhile effort. I've created an issue to capture this as a separate task: https://github.com/keystonejs/keystone/issues/2293

Reply:
OK, good
Then I bring them back for now